### ISSUE TITLE
AMDMemTweak and Other Optimized Timings

### ISSUE THREAD

**Issue by jch9678** [RainbowMiner/RainbowMiner#521]
RM have you been following the developments for changing timings on the fly using AMDMemTweak by eliovp/heliox?  Claymore will be introducing something soon and Doktor83 from srbminer has his own implementation that uses tweaking profiles.  RM could take advantage of srbminer tweaking profiles now, it's just something that needs to be added to the config files of srbminer.  It might be too much to automate that though, but tweaking profile does need to be enabled in the .ps1 I believe.  I've referenced RainbowMiner here  https://github.com/Eliovp/amdmemorytweak/issues/19  If this is able to be implemented then it would be very easy to have RainbowMiner switch between algos with optimized timings...

**Comment by RainbowMiner** [comment]
Ohh! Very nice :) Would you do the testing, if I implement that?

**Comment by jch9678** [comment]
Of course, if you don't mind that I may not be able to test each day.  Also if you could help me out with the powershell crashes.

**Comment by foxwarernd** [comment]
Ready to help with testing with my RX 480 4 Gb, 8Gb

**Comment by ParaplegicRacehorse** [comment]
Any movement on this? Results to post? I admit I understand next to nothing about what the settings actually mean and how they might affect anything. BUT ready to test R9 390X
